### DISCUSSION TITLE
release 0.3.5

### DISCUSSION THREAD

**Discussion by kheina** [kheina-com/Blue-Blocker#229] [General]
@cooljeanius I see that a merge called 0.3.5 was merged to main, should I make a release for this?

**Comment by cooljeanius** [comment]
Yeah I think that was @rougetimelord's intent with the PR

**Reply by kheina** [reply]
ok, I've created a release and sent the zips to both stores. this is the first release I haven't touched so hopefully nothing's busted. I did take a quick look at the diff between 0.3.4 and 0.3.5 and didn't see anything crazy so I think it should be fine

**Reply by rougetimelord** [reply]
Thanks Dani! Crossing my fingers that I didn't bugger anything up :)
